src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/NotificationsControllerTest.java

```diff
@@ -216,23 +214,32 @@ public void shouldNotSendContestedHwfSuccessfulEmail() {
     }
 
     @Test
-    public void givenSolAgreedToEmails_and_noPreviousHearing_shouldSendPrepareForHearingEmail_and_PrintHearingDocuments() {
+    public void givenSolAgreedToEmails_shouldSendPrepareForHearingEmail() {
         when(caseDataService.isApplicantSolicitorAgreeToReceiveEmails(any())).thenReturn(true);
-        when(caseDataService.isContestedPaperApplication(any())).thenReturn(true);
-        when(featureToggleService.isRespondentJourneyEnabled()).thenReturn(true);
-        when(notificationService.shouldEmailRespondentSolicitor(any())).thenReturn(true);
-        when(hearingDocumentService.alreadyHadFirstHearing(any())).thenReturn(false);
+        when(caseDataService.isContestedApplication(any())).thenReturn(true);
+        when(notificationService.shouldEmailRespondentSolicitor(any())).thenReturn(false);
 
         notificationsController.sendPrepareForHearingEmail(AUTH_TOKEN, buildCallbackRequest());
 
         verify(notificationService).sendPrepareForHearingEmailApplicant(any());
-        verify(notificationService).sendPrepareForHearingEmailRespondent(any());
-        verify(hearingDocumentService).sendFormCAndGForBulkPrint(any(), eq(AUTH_TOKEN));
     }
 
+    //    @Test
```

Comment:
Why is this commented out? If it is not valid anymore just remove it, we shouldn't merge commented out code to master